devstats/queries/pr_data_query.gql

```diff
@@ -9,8 +9,17 @@ query {
           state
           title
           createdAt
+          updatedAt
+          closedAt
           baseRefName
           mergeable
+          labels(first: 100) {
```

Comment:
Why the limit here?

Reply:
It was just to match the same limit as for the issue query. There is no real reason for it otherwise.